### PR TITLE
Fix README inconsistencies for Node runtimes

### PR DESCRIPTION
**Pull request by thinkverse**
This PR fixes the inconsistencies of the READMEs for the Node runtimes. Some were out of date with v3, and most if not all had Docker command examples that didn't work. Node 18+ also requires there to be a `package.json` file, whilst earlier versions did work with just an entry point, that is no longer the case. Given this, I opted to update the steps to include a new seconds step to show how to generate one.

### CODE REVIEW

**Comment by thinkverse** [comment]
PHP runtimes error out with: `The job exceeded the maximum log length, and has been terminated.` Build failure was not related to this PR.

**Review by christyjacob4 — APPROVED** [review]

File: runtimes/node-14.5/README.md [hunk 1]
```diff
@@ -18,22 +18,27 @@ module.exports = async (context) => {
...
-docker run -e OPEN_RUNTIMES_ENTRYPOINT=index.js --rm --interactive --tty --volume $PWD:/mnt/code openruntimes/node:v3-14.5 sh helpers/build.sh
+docker run --rm --interactive --tty -v $(pwd):/mnt/code:rw -e OPEN_RUNTIMES_ENTRYPOINT=index.js openruntimes/node:v3-14.5 sh helpers/build.sh "npm install"
 ```
 
-3. Spin-up open-runtime:
+4. Spin-up open-runtime:
 
 ```bash
-docker run -p 3000:3000 -e OPEN_RUNTIMES_SECRET=secret-key --rm --interactive --tty --volume $PWD/code.tar.gz:/mnt/code/code.tar.gz:ro openruntimes/node:v3-14.5 sh helpers/start.sh "pm2 start src/server.js --no-daemon"
+docker run -p 3000:3000 -e OPEN_RUNTIMES_SECRET=secret-key --rm --interactive --tty --volume $(pwd)/code.tar.gz:/mnt/code/code.tar.gz:ro openruntimes/node:v3-14.5 sh helpers/start.sh "pm2 start src/server.js --no-daemon"
```
Comment (Meldiron) [inline runtimes/node-14.5/README.md:38]: Is this change necessary? We do `$PWD` in all other runtime READMEs. If it breaks here, we need to do this change everywhere. If it works without this change, we can revert it from node readmes.

Comment (thinkverse) [reply runtimes/node-14.5/README.md:38]: It is what's written in the [tests.sh](https://github.com/open-runtimes/open-runtimes/blob/main/tests.sh) file, where I got the working docker commands from. And it's also written like that in the [CONTRIBUTING.md](https://github.com/open-runtimes/open-runtimes/blob/main/CONTRIBUTING.md#installation) file. But I'll change it back to `PWD`. 👍

**Review by loks0n — APPROVED** [review]

**Comment by Meldiron** [comment]
Heyy 👋 With recent rework, repository now has many streamlined processes, including ones for runtime READMEs. THere is now only one readme for all versions of a runtime. Closing this PR in favour of: https://github.com/open-runtimes/open-runtimes/pull/264
